docker-java-core/src/main/java/com/github/dockerjava/core/command/ExecStartResultCallback.java

```diff
@@ -14,6 +14,7 @@
  * @author Marcus Linke
  *
  */
+@Deprecated
```

Comment:
Not quite sure why is it marked as @Deprecated. 
I can't find alternative in the API package, could you please advice a replacement?

Reply:
Yes, please use the new generic `ResultCallback.Adapter`